spec/components/middleware/request_tracker_spec.rb

```diff
@@ -386,6 +394,114 @@ def warn(*args)
       status, _ = middleware.call(env2)
       expect(status).to eq(200)
     end
+
+    describe "diagnostic information" do
+      it "is included when the requests-per-10-seconds limit is reached" do
+        global_setting :max_reqs_per_ip_per_10_seconds, 1
+        x = 0
```

Comment:
```suggestion
        called = 0
```

I think this makes it clearer as a variable name.